app.py

```diff
@@ -15,14 +15,26 @@
 @app.route('/api/v1/users/<int:user_id>', methods=['GET'])
 def get_one_user(user_id):
     user = mongo.db.users.find_one_or_404({'_id': user_id})
-    user_plants = list(mongo.db.plants.find({'user_id': user_id}))
     response = {
         'user_id': user['_id'],
-        'user_name': user['name'],
-        'plants': user_plants
+        'user_name': user['name']
     }
     return jsonify({'user': response})
 
+@app.route('/api/v1/users/<int:user_id>/plants', methods=['GET'])
+def get_user_plants(user_id):
+    user = mongo.db.users.find_one_or_404({'_id': user_id})
```

Comment:
Awesome job breaking out this endpoint. It will be helpful to avoid over fetching the data.